STACpopulator/implementations/CMIP6_UofT/collection_config.yml

```diff
@@ -4,4 +4,17 @@ description: Coupled Model Intercomparison Project phase 6
 keywords: ['CMIP', 'CMIP6', 'WCRP', 'Climate Change']
 license: "CC-BY-4.0"
 spatialextent: [-180, -90, 180, 90]
-temporalextent: ['1850-01-01', null]
\ No newline at end of file
+temporalextent: ['1850-01-01', '2500-01-01']
+
+assets:
+  homepage:
+      title : Project homepage
+      href : https://wcrp-cmip.org/cmip-phase-6-cmip6/
+      description: CMIP6 project homepage
+      media_type: text/html
```

Comment:
Seems more appropriate to use `links` with `rel: about`. 
https://www.iana.org/assignments/link-relations/link-relations.xhtml

Reply:
Again it seemed from my examination of your implementation that you were using about like links in assets.

Reply:
@fmigneault was this a misuse too?

Reply:
Yes. This should also be in `links`.